frontend/components/item/value/Base.vue

```diff
@@ -1,6 +1,25 @@
 <template>
   <div v-if="valueToView">
-    <component :is="'item-value-type-' + valueToView.type" :is-user-logged="isUserLogged" :value-to-view="valueToView" :save="editValue" />
+    <v-tooltip v-if="truncated" top open-delay="200">
```

Comment:
I think it's not necessary truncate the strings with the new changes, the text will be wrapped.
I think that wrap the text will be better than truncate the text, we can check it with Charles.